src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -462,6 +462,10 @@ public string NameClaimType
         /// Gets or sets a value indicating whether SAML tokens must have at least one AudienceRestriction.
```

Comment:
Now this flag is used for SAML and JWT tokens. Could you update the main summary comments to reflect this.

Reply:
Yes, updated!